### ISSUE TITLE
Feature Request: Calendar Applet

### ISSUE THREAD

**Issue by benjaminwolkchen** [linuxmint/cinnamon#9224]
```
 * Cinnamon version (cinnamon --version)
   - Please specify if you are using the daily builds PPA
     (https://launchpad.net/~linuxmint-daily-build-team/+archive/ubuntu/daily-builds).
 * Distribution - Mint 19.3
 * Graphics hardware *and* driver used
 * 32 or 64 bit
 * Attach /home/<username>/.xsession-errors, or /var/log/syslog
 ```

**Feature Request**
Calendar integration with gnome-calendar.
If gnome-calendar is installed, there should be an indicator then there is an appointment.
Preferably in the color specified by gnome-calendar in the calendar options.

**Comment by benjaminwolkchen** [comment]
Elementary OS does it this way:
![](https://blog.elementary.io/images/updates-for-october-2019/datetime.png)

**Comment by ManIVIctorious** [comment]
I agree, at the moment the cinnamon calendar applet is the most useless implementation of a calendar I have seen on any desktop. Calling it clock applet would be much more appropriate.

Ideally this wouldn't be dependent on gnome-calendar though. AFAIK one of the missing pieces here is a common, cross-desktop calendar backend solution (e.g. gnome-contacts has libfolks which works cross desktop). Gnome-calendar's backend is  evolution-dataserver. For a gtk desktop that dependency makes the most sense, but ideally one should design this (software wise) with other backends in mind.

**Comment by mfreeman72** [comment]
I've been asking for this feature for YEARS (and apparently asking in all the wrong places). Perhaps, instead of requiring gnome-calendar, make that one of several possible sources? In other words, if the user installed gnome-calendar, have the option to use that as a source, selectable in a menu in the configuration window. Otherwise, allow other options, such as connecting with Thunderbird's calendar addon, or with an online source such as Google Calendar, or with an iCal file at a filesystem location or online URL. As far as appearance, I really like the Elementary OS example above. Otherwise the old Gnome 2 style, with the event list below the calendar would work.

**Comment by haggen88** [comment]
This integration would be very useful !
